electron-app/main/window.ts

```diff
@@ -4,8 +4,7 @@ import MENU from './menu';
 import popupContextMenu from './contextMenu';
 import { APP_TITLE } from '../constants';
 
-const IS_DEV: boolean =
-  process.env.NODE_ENV === 'development' || !!process.env.IS_STAGING || !process.env.NODE_ENV;
+const IS_DEV: boolean = process.env.NODE_ENV === 'development' || !process.env.NODE_ENV;
```

Comment:
We want to declare the constant once for the whole app.
Use `IS_DEV` from `utils/environment`.